static/js/pages/instances/InstanceDetailPanel.tsx

```diff
@@ -86,6 +86,14 @@ const InstanceDetailPanel: FC<Props> = ({ instance, notify, onClose }) => {
               <th>Last used</th>
               <td>{isoTimeToString(instance.last_used_at)}</td>
             </tr>
+            <tr className="p-address-row">
+              <th>Address</th>
```

Comment:
I think we should split into ipv6 and ipv4, as we do in the table. We should probably adopt the overview tab on the instance detail page as well.

Reply:
Also the ipv6 can be fairly long (i.e. fd42:34d1:2187:887a:216:3eff:fe9e:f091), overflowing the width of the panel. Maybe wrap every ipv6 with a `<div className="u-truncate" title={ipv6}>{ipv6}</div>`. The title makes it possible to hover with the mouse and see the full address, while the truncate adds a nice ellipsis when it is too long.

Reply:
> Also the ipv6 can be fairly long (i.e. fd42:34d1:2187:887a:216:3eff:fe9e:f091), overflowing the width of the panel. Maybe wrap every ipv6 with a `<div className="u-truncate" title={ipv6}>{ipv6}</div>`. The title makes it possible to hover with the mouse and see the full address, while the truncate adds a nice ellipsis when it is too long.

Honestly, I don't think this is a good idea for IP addresses. The reason is: the most common use case, at least in my opinion, is **copy-pasting** the address. If you put it into a `title`, you cannot copy it anymore.
My proposal is: we keep it as it is now, which is not so bad since it is already wrapping on the ":" (so every 4 chars at most), and we add the title. So you still get a way to see the full address (including the few characters overflowing the width) and a way to copy-paste the address. What do you think?

Reply:
In chrome, it doesn't wrap, it looks a bit odd for me:

![Screenshot from 2023-01-13 09-42-33](https://user-images.githubusercontent.com/1155472/212276455-b21cf77c-00b3-4ccf-8f92-5a07459b6410.png)

Reply:
It even could wrap in a very unfortunate way, so you see only part of an address and think it is complete.

Reply:
It's wrapping on the colon, as I said. You are missing only 3 characters. It looks odd, but be sure that as soon as you ellipsise it you will be bombarded by requests of angry users that want it to be copy-pastable. So another solution needs to be found in my opinion.

Reply:
Maybe truncate and add a copy button next to it?

Reply:
We have `p-icon--copy`

Reply:
Good idea, I'll try that!

Reply:
Yeah, it is very neat. I like it.